framework_tvb/tvb/core/services/import_service.py

```diff
@@ -363,16 +366,15 @@ def import_project_operations(self, project, import_path):
         return imported_operations
 
     @staticmethod
-    def _get_param_from_view_model_gid(view_model):
-        return '{"gid": "' + view_model.gid.hex + '"}'
-
-    @staticmethod
-    def _get_new_form_view_model(operation):
+    def _get_new_form_view_model(operation, xml_parameters):
         # type (Operation) -> ViewModel
         ad = ABCAdapter.build_adapter(operation.algorithm)
         view_model = ad.get_view_model_class()()
-        if operation.parameters:
-            params = json.loads(operation.parameters)
+        operation.view_model_gid = view_model.gid.hex
+        dao.store_entity(operation)
```

Comment:
The operation view_model_gid is updated here, but it is true that the method that contains this is in a for, but that is the ideea to update all operation view_model_gid's, because at initial creation they are all None/Null singe there is no view_model then

Reply:
I still think the same operation is submitted for storage twice in DB, in an unnecessary manner.